### ISSUE TITLE
A11y_30days_Welcome to 30days of learning_Keyboard: After navigating through the “Submenu of Support button” when press tab keyboard focus is not visible, when press tab again then keyboard focus is visible on “Blog link”.

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/30daysof#173]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yTCS;#A11yMAS;#A11ySev3;#30days;#P3_30days_Dec22;#WCAG2.1.1;#DesktopWeb;#Win11;ChromiumEdge

### Environment Details:
Application Name: 30days
#URL: https://microsoft.github.io/30daysof/ 
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)
### Repro Steps:
1.	Hit the URL: https://microsoft.github.io/30daysof/
2.	“30Days of Learning” page will open.
3.	Tab to “Support button” in header section and press enter.
4.	Tab to “Progressive web apps link” in submenu and press tab again.
5.	Observe that keyboard focus is not visible on the “Blog” link.
### Actual:
After navigating through the “Submenu of Support button” keyboard focus is not visible and when tab again keyboard focus is visible on “Blog” link.
### Expected:
After navigating through the “Submenu of Support button” keyboard focus should be visible on “Blog” link directly.

### User Impact:
Keyboard only users will face difficulty to understand where the keyboard focus is.

### Attachment:
[209326939-67e06357-9ca7-4f42-8c32-aae2ba924976.webm](https://user-images.githubusercontent.com/85863895/211751648-99be4f1b-76ce-4626-a1a6-f9dc5512debe.webm)

**Comment by nitya** [comment]
Navbar sub-menus no longer exist - moved to footer as flat hierarchy

**Comment by nitya** [comment]
@Ashutoshsahoo164 can you verify if issue is resolved?

**Comment by Raghu-addepalli** [comment]
![image](https://user-images.githubusercontent.com/91586743/220275547-cb8d4d8c-dc8e-459a-84eb-6ac9998e86f4.png)
verified the issue in below https://microsoft.github.io/30daysof/ environment the controls causing the issue are not available in environment hence closing the bug and added attachment.
Githubtags:  #Closed;
